src/scanners/css.js

```diff
@@ -26,7 +26,15 @@ export default class CSSScanner extends BaseScanner {
                 }
 
                 this.validatorMessages = this.validatorMessages.concat(
-                  rules[cssRule](rule));
```

Comment:
Even better would be something without the word "rule" entirely.

`rule` stays `rule` and `cssRule`/`cssRuleFunc` becomes `cssInstruction` or even `cssCode` perhaps? Getting the word "Rule" out of there entirely would really help scalability even if the alternate names are a bit weird.